### DISCUSSION TITLE
September 25th Team Check-in

### DISCUSSION THREAD

**Discussion by Gersh305** [ecn310/course-project-zipcentercrime#7] [General]
# For Each Member: #
- Something someone did to help the team this week:
- Something I could have done better for the team this week:
- An area / suggestion for team improvement:

**Comment by Gersh305** [comment]
# Leo #
- Something someone did to help the team this week: Sophia helped the team by working on the second paper summary early; Rachel helped by emailing Professor Deza.
- Something I could have done better for the team this week: I could have written more in depth team meeting summaries.
- An area / suggestion for team improvement: We could have our mediator or communicator be in charge of assigning everyone things to do to keep everyone on top of their assignments.

**Comment by regaudre** [comment]
## Rachel 
- Something someone did to help the team this week: Sophia for taking the lead and finishing the second paper summary!
 - Something I could have done better for the team this week: I definitely could've communicated my available time to work on this project this week better. A lot of this issue arose because I was sick, so my goal became to survive the week and get the bare minimum done instead of putting real work into different things. To improve on this I need to get in a better routine with my time that I can fall back on even if I do not feel well.
 - An area / Suggestion for team improvement: Better communication about what tasks need to be immediately accomplished, who is working on those tasks, and when our goal is to have them done by.

**Comment by sortizh** [comment]
# Sophia
- Something someone did to help the team this week: Wes changed the second paper summary file to .md. Leo found some data on drug treatment centers. Rachel emailed Professor Deza
- Something I could have done better for the team this week: Making sure we are all aware of the mid week deadlines
- An area/suggestion for team improvement: After every class if there is a new deadline, we should put it in our group chat so we all know and are aware, as well as delegating tasks better

**Comment by Wes-Mac** [comment]
# Weston 
___
- Something someone did to help the team this week: Sophia uploaded rough drafts of the reading summaries for the team to build on, Rachel reached out to Professor Deza, and Leo completed our group logs.
-  As the timekeeper I need to step up to make sure we reach our deadlines as a team
- We need to work on meeting deadlines as a team and I will begin to send more reminders to our group chat
